model/trace/http.yaml

```diff
@@ -120,12 +121,9 @@ groups:
           - Port identifier of the [request target](https://www.rfc-editor.org/rfc/rfc9110.html#target.resource)
             if it's sent in absolute-form.
           - Port identifier of the `Host` header
-      - ref: server.socket.address
+      - ref: server.ip
         requirement_level: opt_in
-        brief: Local socket address. Useful in case of a multi-IP host.
-      - ref: server.socket.port
-        requirement_level: opt_in
-        brief: Local socket port. Useful in case of a multi-port host.
```

Comment:
same here, it's not clear to me whether we need a replacement for `server.socket.port`?